File: Hrm.Domain/OfficeOrder.cs
using Hrm.Domain.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hrm.Domain
{
    public class OfficeOrder : BaseDomainEntity
    {
        public int Id { get; set; }
        public int? EmpId { get; set; }
        public int? OrderTypeId { get; set; }
        public int? OfficeId { get; set; }
        public int? DepartmentId { get; set; }
        public int? SectionId { get; set; }
        public int? DesignationId { get; set; }
        public DateOnly? OrderDate { get; set; }
        public string? OrderNo { get; set; }
        public string? FileUrl { get; set; }
        public string? Remark { get; set; }
        public bool? IsActive { get; set; }
        public int? MenuPosition { get; set; }

        public virtual EmpBasicInfo? EmpBasicInfo { get; set; }
        public virtual OrderType? OrderType { get; set; }
        public virtual Office? Office { get; set; }
        public virtual Department? Department { get; set; }
        public virtual Section? Section { get; set; }
        public virtual DesignationSetup? DesignationSetup { get; set; }
    }
}

File: ClientApp/src/app/views/notifications/models/office-order.ts
export class OfficeOrder {
    id: number = 0;
    empId: number | null = null;
    orderTypeId: number | null = null;
    officeId: number | null = null;
    departmentId: number | null = null;
    sectionId: number | null = null;
    designationId: number | null = null;
    orderDate: Date = new Date;
    orderNo: string = "";
    orderFile: File | null = null;
    fileUrl: string = "";
    remark: string = "";
    isActive: boolean = true;
    menuPosition: number | null = null;

    orderTypeName: string = "";
    officeName: string = "";
    departmentName: string = "";
    sectionName: string = "";
    designationName: string = "";
}
